### ISSUE TITLE
Template scripts after exam mode

### ISSUE THREAD

**Issue by quentinguidee** [Omega-Numworks/Omega#315]
#### What I want to see in the next version of Omega
When the user leave the exam mode, put back the template scripts.

**Comment by M4xi1m3** [comment]
This is the kind of issues where when you first take a look at it you're like "meh, easy". The more I look at it, the more I think it's impossible without a heck tone of modifications to the current app system.

**Comment by LeGmask** [comment]
Duplicate of #174

**Comment by RedGl0w** [comment]
> Duplicate of #174

No, that's not a duplicate, because template scripts != users' script + template scripts

**Comment by quentinguidee** [comment]
:troll:

**Comment by LeGmask** [comment]
the difference isn't really big 😅

**Comment by RedGl0w** [comment]
It is big enough for having two different issues (technically this one is easier I think)

**Comment by LeGmask** [comment]
do it then :troll:
